tutorials/basic-topics/linear-search.md

```diff
@@ -0,0 +1,139 @@
+---
+title: 'Linear Search'
+description: 'Linear search is a searching technique that works in a linear/sequential fashion. It traverses whole of the data structure and returns the position(s) where the value is found. If not found then it simple returns nothing.'
+hide_table_of_contents: true
+---
+
+<TutorialAuthors names="@siddoinghisjob"/>
+
+## Overview
+
+Lets say we have a linear data structure - array, linked list - and we need to search for a certain element. To search there are mainly two searching techniques, one being linear search and the other is [binary search](./binary-search). In linear search we traverse the whole array and then while traversing we check for the particular item. If there's a match then we return that position (we can return multiple positions too by using an array or vector or by simply printing all the postions).
```

Comment:
1. Actually there are more than two searching techniques. I don't think you need to mention that at all. Please delete that.

2.  This part is quite unclean. Please elaborate more.

  > we can return multiple positions too by using an array or vector or by simply printing all the postions

3. Add some general steps

4.  Include time complexity for linear search (best / worst / avg cases).